frontend/src/components/Forms/ShoutoutsPage/ShoutoutCard.tsx

```diff
@@ -1,35 +1,17 @@
 import { Card } from 'semantic-ui-react';
-import { Member } from '../../../API/MembersAPI';
+import { Shoutout } from '../../../API/ShoutoutsAPI';
 
-type Props =
-  | {
-      readonly giver: Member;
-      readonly receiver: Member;
-      readonly message: string;
-      readonly isAnon: false;
-    }
-  | {
-      readonly receiver: Member;
-      readonly message: string;
-      readonly isAnon: true;
-    };
+const ShoutoutCard = (props: Shoutout): JSX.Element => {
+  const { giver, receiver, message, isAnon } = props;
 
-const ShoutoutCard = (props: Props): JSX.Element => {
-  const { receiver, message } = props;
-
-  let fromString;
-  if ('giver' in props) {
-    const { giver } = props;
+  let fromString = 'From: Anonymous';
+  if (!isAnon) {
     fromString = `From: ${giver?.firstName} ${giver?.lastName} (${giver.email})`;
```

Comment:
It's not a big deal, but this could be done with a tertiary operator so that `fromString` can be a const. 
```suggestion
  const fromString = isAnon ? 'From: Anonymous' :  `From: ${giver?.firstName} ${giver?.lastName} (${giver.email})` ;
```